### ISSUE TITLE
Cannot Lint on ST3

### ISSUE THREAD

**Issue by arusahni** [biermeester/Pylinter#30]
Pylinter was working until I restarted ST just now.  I now get an exception on load, and whenever I try to lint manually I get the following exception:

```
Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 549, in run_
    return self.run(edit)
  File "/home/aru/.config/sublime-text-3/Packages/Pylinter/pylinter.py", line 194, in run
    settings = PylSet.read_settings()
  File "/home/aru/.config/sublime-text-3/Packages/Pylinter/pylinter.py", line 90, in read_settings
    ignore = [t.lower() for t in cls.get_or('ignore', [])]
TypeError: 'NoneType' object is not iterable
```

**Comment by biermeester** [comment]
This error is really making me suspect there's something wrong with the Pylinter you installed. Please remove it and re-install. If the problem keeps popping up, please refer to my comment to the other issue you posted.

**Comment by arusahni** [comment]
I'm feeling dumb. I renamed one of my virtualenvs, but didn't change the pythonpath setting in my project file. Sorry for the false positive!
